react-app/components/DropZoneFile/index.jsx

```diff
@@ -19,43 +20,68 @@ const fileTypesAllowed = {
 const DropZoneFile = () => {
   const { setFilesToUploadList } = useContext(GalleryContext);
 
+  async function uploadAndFetchFiles(formData, _ = 10) {
+    try {
+      const { element: globalElement, xblockId } = globalObject;
+      const fileUploadHandler = globalObject.runtime.handlerUrl(globalElement, 'file_upload');
+
+      // Upload the file
+      const uploadResponse = await apiConfig.post(fileUploadHandler, formData, {
+        headers: {
+          'Content-Type': 'multipart/form-data'
+        }
+      });
+
+      if (uploadResponse.status === 200) {
```

Comment:
can we use constants for the status codes? what if it fails? can we show an error message?